src/com/pentalog/pentastagiu/homework2/shop/ClientsOfShop.java

```diff
@@ -0,0 +1,42 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ClientsOfShop {
+
+    private String name;
+    private MembershipStatus membershipStatus;
+
+    public ClientsOfShop(String name, MembershipStatus membershipStatus) {
+        this.name = name;
+        this.membershipStatus = membershipStatus;
+    }
+
+    public int priceForCostumer(){
```

Comment:
o metoda trebuie sa contina un verb pentru a sugera ce anume se intampla in logica dinauntrul ei, mai potrivit ar fi getDiscount/getDiscountBasedOnMembership